modules/extensions-spring-support/src/test/resources/transactional-parameters-injection-config.xml

```diff
@@ -24,7 +24,7 @@
     </flow>
 
     <flow name="injectInOperationJoinNotSupported">
-        <tx:inject-transactional-action config-ref="localTx1" action="NOT_SUPPORTED"/>
+        <tx:inject-transactional-action config-ref="localTx1" transactionalAction="NOT_SUPPORTED"/>
```

Comment:
why does this changes name? Wasn't transactionalAction it's actual name?

Reply:
the name must be always `transactionalAction` the bug I fixed now fixes the name to be always `transactionalAction` doesn't matter how is defined in the Java Code